cmd/jaeger/internal/extension/jaegerstorage/extension.go

```diff
@@ -83,6 +83,20 @@ func GetTraceStoreFactory(name string, host component.Host) (tracestore.Factory,
 	return v1adapter.NewFactory(f), nil
 }
 
+func GetSamplingStoreFactory(name string, host component.Host) (storage.SamplingStoreFactory, error) {
+	f, err := GetStorageFactory(name, host)
+	if err != nil {
+		return nil, err
+	}
+
+	ssf, ok := f.(storage.SamplingStoreFactory)
+	if !ok {
+		return nil, errors.New("storage does not support sampling store")
```

Comment:
Please include name in the error 
```suggestion
		return nil, errors.New("storage does not support sampling store")
```